include/wsl/winadapter.h

```diff
@@ -3,419 +3,4 @@
 
 #pragma once
 
-// These #defines prevent the idl-generated headers from trying to include
```

Comment:
What do you think about including `<basetsd.h>` from here? That way these definitions are present even if `rpcndr.h` isn't included.

Reply:
`<unknwn.h>` includes  `<basetsd.h>` already,
without incude of `<unknwn.h>`, mingw will complain, and  `<basetsd.h>` lack a lot things in `mingw/win32`, I choose `basetsd.h` because I don't want use `windows.h`, there is too much things in `windows.h`.

`<basetsd.h>` is always included by `rpcndr.h`, and `rpcndr.h` always included by `<unknwn.h>`

Reply:
Oh I see. I missed that include. This is probably fine then.